### ISSUE TITLE
NVDA is not announcing the role as 'Menu items' for menu controls.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#16366]
### Steps to reproduce:
1.Open https://developer.microsoft.com/en-us/fluentui#/controls/web/button.
2.Navigate to 'Contextual Menu' button.
3.Expand 'Menu' button.
4.Navigate through the menu items.
5.Observe the issue.

### Actual behavior:

<html>
<body>

NVDA does not announces role for menu items.

NVDA just announces name and position, when user navigates to the menu items.

</body>
</html>

### Expected behavior:

NVDA should be announce the role as well along with controls information as 'Menu item'.

### NVDA logs, crash dumps and other attachments:

https://github.com/nvaccess/nvda/assets/93735775/7ec95de8-c056-4df9-b999-1f80dece70c2

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NA

#### If NVDA add-ons are disabled, is your problem still occurring?
NA

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
NA
